File: utilities/src/main/kotlin/lunalauf/rms/utilities/network/communication/message/request/MinigameRecordRequest.java
package lunalauf.rms.utilities.network.communication.message.request;

import com.google.gson.annotations.SerializedName;
import lunalauf.rms.utilities.network.communication.message.type.RequestType;

public class MinigameRecordRequest extends Request {

    @SerializedName("runner-id")
    public long runnerId;
    @SerializedName("minigame-id")
    public int minigameId;
    @SerializedName("points")
    public int points;

    public MinigameRecordRequest() {
        super(RequestType.MINIGAME_RECORD);
    }
}

File: utilities/src/main/kotlin/lunalauf/rms/utilities/network/communication/message/request/MinigameRecordRequest.kt
package lunalauf.rms.utilities.network.communication.message.request

import com.google.gson.annotations.SerializedName
import lunalauf.rms.utilities.network.communication.message.type.RequestType

class MinigameRecordRequest : Request(RequestType.MINIGAME_RECORD) {
    @SerializedName("runner-id")
    var runnerId: Long = 0

    @SerializedName("minigame-id")
    var minigameId = 0

    @SerializedName("points")
    var points = 0
}
